packages/kbn-test/src/auth/sesson_manager.test.ts

```diff
@@ -91,10 +92,26 @@ describe('SamlSessionManager', () => {
       expect(createCloudSAMLSessionMock.mock.calls).toHaveLength(0);
     });
 
-    test(`'getUserData' should return the correct email & fullname`, async () => {
+    test(`'getEmail' return the correct email`, async () => {
       const samlSessionManager = new SamlSessionManager(samlSessionManagerOptions);
-      const data = await samlSessionManager.getUserData(roleViewer);
-      expect(data).toEqual({ email, fullname });
+      const email = await samlSessionManager.getEmail(roleEditor);
+      expect(email).toBe(testEmail);
+    });
+
+    test(`'getUserData' should call security API and return user profile data`, async () => {
+      const testData: UserProfile = {
+        username: '6ta90xc',
```

Comment:
Is this a hashed value or smth?

Reply:
Cloud generates unique string so I just put a random one. The most important to show it is not related to role/email/full name